packages/app-config-writer/package.json

```diff
@@ -29,6 +29,7 @@
         "!dist/**/*.map"
     ],
     "dependencies": {
+        "@sap-ux/preview-middleware": "workspace:*",
```

Comment:
This should be a `devDep` right? You only need some types.

Reply:
adjusted in d7e5e5c1bed316c6774e4ed3d2389ae03b8f05e2 and c43b05a9b346c5d4498b14828d17d8bfa81d99c3